code/modules/medical/genetics/bioEffects/powers.dm

```diff
@@ -2302,6 +2302,99 @@
 						if (thrown_limb)
 							thrown_limb.throwforce = tmp_force
 
+
+/datum/bioEffect/power/critter/peck
+	name = "Aviornis Rostriformis "
+	desc = "Generates a hardened keratin area between the mouth and nose."
+	id = "beak_peck"
+	msgGain = "You feel your mouth and nose feel more difficult to move."
+	msgLose = "You feel your face return to normal."
+	cooldown = 10 SECONDS
+	occur_in_genepools = 0
+	stability_loss = 15
+	ability_path = /datum/targetable/geneticsAbility/wrapper/peck
+	var/color = "#888888"
+
+/datum/targetable/geneticsAbility/wrapper/peck
+	wrapped_ability = /datum/targetable/critter/peck
+	start_on_cooldown = TRUE
+	has_misfire = FALSE
+	needs_hands = FALSE
+
+/datum/bioEffect/power/critter/snake_bite
+	name = "Ophidentis Vipernox"
+	desc = "Generates an enhanced structure of your fangs allowing for venom."
+	id = "snake_bite"
+	msgGain = "You become oddly aware of your canines and they feel different."
+	msgLose = "You feel less aware of your teeth."
+	cooldown = 20 SECONDS
+	occur_in_genepools = 0
+	stability_loss = 15
+	ability_path = /datum/targetable/geneticsAbility/wrapper/snake_bite
+	var/color = "#888888"
+
+/datum/targetable/geneticsAbility/wrapper/snake_bite
+	wrapped_ability = /datum/targetable/critter/wasp_sting/snake_bite
+	start_on_cooldown = TRUE
+	has_misfire = FALSE
+	needs_hands = FALSE
+	override_params = list("amt1"=5)
+
+
+/datum/bioEffect/power/critter/scorpion_sting
+	name = "Scorpiocauda Vipernox"
+	desc = "Generates a hardened keratin area between the mouth and nose."
```

Comment:
You forgot to add a description while copy-pasting this from beak.

Reply:
🤦 I made the PR, noticed this and thought I addressed it.... just didn't save the file. 😢